ids/set.go

```diff
@@ -19,6 +19,13 @@ const (
 // Set is a set of IDs
 type Set map[ID]bool
 
+// New returns a set of a given size
+func New(size int) *Set {
+	newSet := &Set{}
```

Comment:
This can be:
`newSet := make(Set, size)`

Reply:
Yeah! Because we have the `init` method below I thought I would use it instead of overriding/duplicating its behaviour - does that make sense?

Reply:
I don't think adding this function makes a ton of sense. Rather than calling `ids.New(size)` (which isn't particularly descriptive to someone reading the code) someone could just write `make(ids.Set, size)`.